cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -113,11 +114,22 @@ ScanUnifierNode::~ScanUnifierNode()
 void ScanUnifierNode::getParams()
 {
   std::vector<std::string> topicList;
+  bool point_cloud;
```

Comment:
get rid of this local helper variable
you can directly write `config_.publish_pointcloud` in both if and else case respectively